.changeset/pretty-toes-tan.md

```diff
@@ -0,0 +1,31 @@
+---
+'@leafygreen-ui/text-area': major
+---
+
+[LG-4144](https://jira.mongodb.org/browse/LG-4144)
+
+1. `FormField` styling changes apply to `TextArea`. [See style changes here](https://github.com/mongodb/leafygreen-ui/blob/main/packages/form-field/CHANGELOG.md#102)
+
+2. A default `errorMessage` of `'This input needs your attention'` will render below text area when state is invalid.
+
+3. A default `successMessage` of `'Success'` will render when state is valid. `successMessage` prop accepts a custom string.
```

Comment:
`successMessage` didn't exist before did it?

Reply:
yes new spec from design